### ISSUE TITLE
Entrance.service isn't installed to systemd directories

### ISSUE THREAD

**Issue by SolarAquarion** [Obsidian-StudiosInc/entrance#10]

**Comment by wltjr** [comment]
I some what left that up to packagers, but I guess I can see about having meson install. Should be a standard location. I am not a systemd user so hard to test, etc. Does it work otherwise under systemd? Think some others had issues there. I will see about addressing ASAP. Thanks for the report!

**Comment by SolarAquarion** [comment]
/usr/lib/systemd/system/ 

That's where arch does it.  Probably Fedora etc.  There could be a "systemdddir" switch to set  i suppose

**Comment by wltjr** [comment]
Thanks, I will check it out, I am pretty sure it should be the same for all. That was part of the idea behind systemd, standardization :)

**Comment by SolarAquarion** [comment]
lol

**Comment by wltjr** [comment]
Your not even kidding, this is crazy...
https://www.freedesktop.org/software/systemd/man/systemd.unit.html

I guess any of those directories... That sure does not seem like standardization or making things better. Init scripts always went into a single directory in a single location. That there are system and user, in addition to the different locations...

Safe to say we are not looking to put this in a user subdir. I would think /etc/ to be the proper location. I assume the /usr/lib is for shared /usr. I am some what confused as I thought they moved everything to under /usr.

Anyway likely a trivial fix once we figure out what directory... :P

**Comment by SolarAquarion** [comment]
well, /etc or /usr/lib works

**Comment by wltjr** [comment]
I think I will go with your suggested /usr/lib. Seems like /etc is proper, but I feel like they are merging stuff into /usr so that maybe safest. /usr will always be there, not sure about /etc :)

This clarifies it a bit
https://unix.stackexchange.com/questions/206315/what-is-difference-between-usr-lib-and-etc-systemd

Not sure what category this falls. I guess since not coming as part of distribution it goes in /etc. I guess to separate out base system as provided by upstream and anything an admin has installed.

I am good with either. Which do you think?

**Comment by wltjr** [comment]
Ok there you go. Turns out I had the block already just under has_pam vs has_logind. There is no code written for logind/elogind. I assume this works under systemd, no clue. Past problems I think were due to use of sudo which I got rid of. Maybe that was needed for systemd, or not. I need to setup some travis tests builds, with other options. Though not sure I can start entrance under travis. Do not care about an UI, logs would be enough. Likely have to break down and setup another VM. Been on my todo list for over a year. Still bitter I lost a 1TB spinning disk that was like 1yr old. Last spinning disk drive I had and of course where I had my debian and centos dev vms. Next time will do one for arch, as I think that should be the main 3. Also need to do FreeBSD. Though if I go that route I may switch to FreeBSD myself. Its what I would run if I stopped running Gentoo and was not using say Funtoo. Love the FreeBSD Foundation and the organization all around.
